recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,105 @@
+import os, shutil
+from conan.tools.cmake import CMake, CMakeToolchain
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.39.0"
+
+class ConanFile(ConanFile):
+    name = "tbb"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/oneapi-src/oneTBB"
+    description = """oneAPI Threading Building Blocks (oneTBB) lets you easily write parallel C++
+programs that take full advantage of multicore performance, that are portable, composable
+and have future-proof scalability."""
```

Comment:
Using `"""` will introduce newlines.
```suggestion
    description = "oneAPI Threading Building Blocks (oneTBB) lets you easily write parallel C++" \
        "programs that take full advantage of multicore performance, that are portable, composable" \
        "and have future-proof scalability."
```